frontend/public/components/dashboards-page/dashboards.tsx

```diff
@@ -1,20 +1,47 @@
 import * as React from 'react';
 import { RouteComponentProps } from 'react-router-dom';
 
+import * as plugins from '../../plugins';
 import { OverviewDashboard } from './overview-dashboard/overview-dashboard';
-import { HorizontalNav, PageHeading } from '../utils';
+import { HorizontalNav, PageHeading, Page } from '../utils';
+import { Dashboard } from '../dashboard/dashboard';
+import { DashboardGrid, GridPosition } from '../dashboard/grid';
+import { DashboardsCard } from '@console/plugin-sdk';
 
-const tabs = [
+const tabs: Page[] = [
   {
     href: '',
     name: 'Overview',
     component: OverviewDashboard,
   },
 ];
 
+const getCardsOnPosition = (cards: DashboardsCard[], position: GridPosition): React.ComponentType<any>[] =>
```

Comment:
To render lazy (async) components, you can use Promise-based `AsyncComponent`.

Reply:
Using `AsyncComponent` now